app/gateway/changelog.md

```diff
@@ -624,6 +624,55 @@ In some environments (such as ARM64-based environments), the default value is in
 The plugin can erroneously block valid requests if any of the parameters continue with the default values.
 To mitigate this issue, configure the JSON Threat Protection plugin with limits for all of the `max_*` parameters.
 
+
+## 3.7.1.3
```

Comment:
There's two 404s for Debian blog posts in the changelog. I think we can probably just remove those two links.